packages/ui/src/views/repo/webhooks/webhook-create/components/create-webhooks-form-fields.tsx

```diff
@@ -26,54 +26,45 @@ export const WebhookToggleField: FC<WebhookFormFieldProps> = ({ register, watch,
 )
 
 export const WebhookNameField: FC<WebhookFormFieldProps & { disabled?: boolean }> = ({ register, disabled, t }) => (
-  <ControlGroup>
-    <FormInput.Text
-      id="name"
-      {...register!('identifier')}
-      placeholder="Name your webhook"
-      autoFocus
-      disabled={disabled}
-      label={t('views:repos.name', 'Name')}
-    />
-  </ControlGroup>
+  <FormInput.Text
+    id="name"
+    {...register!('identifier')}
+    placeholder="Name your webhook"
+    autoFocus
+    disabled={disabled}
+    label={t('views:repos.name', 'Name')}
+  />
 )
 
-export const WebhookDescriptionField: FC<WebhookFormFieldProps> = ({ register, errors, t }) => (
-  <ControlGroup>
-    <Textarea
-      id="description"
-      {...register!('description')}
-      placeholder={t('views:repos.descriptionPlaceholder', 'Enter a description')}
-      label={t('views:repos.description', 'Description')}
-      error={errors?.description?.message?.toString()}
-    />
-  </ControlGroup>
+export const WebhookDescriptionField: FC<WebhookFormFieldProps> = ({ register, t }) => (
+  <FormInput.Textarea
+    id="description"
+    {...register!('description')}
+    placeholder={t('views:repos.descriptionPlaceholder', 'Enter a description')}
+    label={t('views:repos.description', 'Description')}
+  />
 )
 
 export const WebhookPayloadUrlField: FC<WebhookFormFieldProps> = ({ register, t }) => (
-  <ControlGroup>
-    <FormInput.Text
-      autoComplete="new-password"
-      data-form-type="other"
-      id="payloadUrl"
-      {...register!('url')}
-      placeholder={t('views:repos.urlPlaceholder', 'https://example.com/harness')}
-      label={t('views:repos.urlLabel', 'Payload URL')}
-    />
-  </ControlGroup>
+  <FormInput.Text
+    autoComplete="new-password"
```

Comment:
Nothing to do with this PR, but this seems off here. Stating it should auto complete as a new password will trigger the browser to suggest a new password when this field is actually a URL field. Not sure why we'd want that. Same with other fields; the autoComplete values seem totally incorrect from my reading

Reply:
Should I address this issue in this PR? Tbh, I don't know the context of this part of the app🙇‍♂️